Holy_locket.WebAPI/Controllers/GetDoctorsController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Holy_locket.WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GetDoctorsController : ControllerBase
+    {
+        public GetDoctorsController() 
+        { 
+        }
+
+        [HttpGet]
+        public IActionResult GetDoctors() 
+        {
+            try {
+                //получение данных
```

Comment:
don't push explanation comments to the remote repo. If you need to explain your code - it's a bad code. You can add to-do points for future work.
```suggestion
                //ToDo: add getting data logic from DoctorsService
```